File: src/VetrinaDolci.WebAPI/Models/Ingrediente.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace VetrinaDolci.WebAPI.Models
{
    public class Ingrediente
    {
        public int Id { get; set; }
        public string Nome { get; set; }
        public double? Proteine { get; set; }
        public double? Zuccheri { get; set; }
        public double? Grassi { get; set; }
        public double? Colesterolo { get; set; }
        public double? Fibra { get; set; }
        public double? Kcal { get; set; }

        public List<IngredientiDolce> IngredientiDolce { get; set; }
    }
}

File: src/vetrina-dolci-client/src/app/models/ingrediente.ts
import { IngredientiDolce } from "./ingredienti-dolce";

export class Ingrediente {
    id: number;
    nome: string;
    Proteine: number;
    Zuccheri: number;
    Grassi: number;
    Colesterolo: number;
    Fibra: number;
    Kcal: number;

    ingredientiDolce: IngredientiDolce[]

    constructor(data?: Partial<Ingrediente>) {
      Object.assign(this, data);
    }
  }
  